### DISCUSSION TITLE
can't find ForeignKey in Column Types!

### DISCUSSION THREAD

**Discussion by chaochaokeji** [nocodb/nocodb#1454] [Q&A]
i want to belongs to table. but i can't find ForeignKey when set Column Types!

**Comment by dstala** [comment]
@chaochaokeji 
Refer documentation [here](https://docs.nocodb.com/setup-and-usages/link-to-another-record)

**Reply by james-innes** [reply]
The [ForeignKey](https://docs.nocodb.com/#foreignkey) column typed used to be present and is littered over the codebase. I can see at [some point](https://github.com/nocodb/nocodb/blob/f1cd14192285c865b01f983a8dfcd8a83a76493b/packages/nc-gui/components/project/spreadsheet/helpers/uiTypes.js#L13) it was comment out. I understand I can still create this relationship but it is not an enforced constraint. I have to manually add all the related rows which is not feasible. Why was it removed and the docs not updated and how can I create a traditional One to Many relationship in NocoDB please?
